server/set-port.js

```diff
@@ -15,14 +15,25 @@
 require('dotenv').config();
 
 const fse = require('fs-extra');
+const portfinder = require('portfinder');
 
-if (process.env.ASSET_COMPUTE_DEV_PORT) {
-    // read/process package.json
-    const file = '../client/package.json';
-    const pkg = fse.readJSONSync(file, { throws: false });
-    // at this point you should have access to your ENV vars
-    pkg.proxy = `http://localhost:${process.env.ASSET_COMPUTE_DEV_PORT}`;
+const portRange = {
+    port: process.env.ASSET_COMPUTE_DEV_PORT || 3000, // minimum port
```

Comment:
`ASSET_COMPUTE_DEV_PORT`  environment variable was supposed to be for the backend port. See description in readme here: https://github.com/adobe/asset-compute-devtool#development

this environment variable should not be used for the client port

Reply:
ok I'll remove this